### PR TITLE
fix(readme): fix .env by using dotenv package

### PR DESCRIPTION
**Pull request by hage1005**
use load_env()

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed
- [ ] Branch name follows `type/descript` (e.g. `feature/add-llm-agents`)
- [ ] Ready for code review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +52,6 @@ cp .env.template .env
 ```
 2. Fill the required environment variables in the `.env` file.
 3. Source the `.env` file.
-```bash
```
Comment (lwaekfjlk) [inline README.md:55]: why this is not required? You are not using .env file now?

Comment (ZhitongGuo) [reply README.md:55]: `load_dotenv()` added in client.py will take care of this

Comment (hage1005) [reply README.md:55]: `load_dotenv()` added in client.py will take care of this

File: research_town/envs/env_proposal_writing_with_rag.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def on_enter(self, **context: Any) -> None:
             context['leader'] = self.agent_manager.sample_leader()
         if 'members' not in context or context['members'] is None:
             context['members'] = self.agent_manager.sample_members()
-        
+
```
Comment (ZhitongGuo) [inline research_town/envs/env_proposal_writing_with_rag.py:36]: added by pre-commit

Comment (hage1005) [inline research_town/envs/env_proposal_writing_with_rag.py:36]: added by precommit

**Pull request merged**